<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Caladea</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GENERADOR DE ARCHIVOS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="genera_excel">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>740</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Generar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="atras_excel">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>740</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;Atr치s</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="qtab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>1121</width>
     <height>641</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tabla">
    <attribute name="title">
     <string>Tabla</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>1101</width>
       <height>531</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="obtener_info">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Obtener informaci칩n</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>344</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Selecciona la tabla:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="carga_excel">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Abrir Tabla</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="imagen">
    <attribute name="title">
     <string>Imagen</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Configuraci칩n del texto</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>479</width>
       <height>117</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>58</number>
      </property>
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>16777198</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="color_picker" resource="color_picker.qrc">
          <normaloff>:/prefijoNuevo/601137-200.png</normaloff>:/prefijoNuevo/601137-200.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Elije el tipo de fuente:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Color de la fuente:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_font">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>FONT_HERSHEY_SIMPLEX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FONT_HERSHEY_PLAIN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FONT_HERSHEY_DUPLEX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FONT_HERSHEY_COMPLEX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FONT_HERSHEY_TRIPLEX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FONT_HERSHEY_COMPLEX_SMALL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FONT_HERSHEY_SCRIPT_SIMPLEX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FONT_HERSHEY_SCRIPT_COMPLEX</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(119, 118, 123);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_fontsize"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tama침o de la fuente:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>681</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Selecciona la imagen:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="carga_imagen">
          <property name="text">
           <string>Carga imagen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
        <property name="spacing">
         <number>37</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Selecciona los campos a colocar:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CheckableComboBox" name="comboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>91</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>28</number>
        </property>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Selecciona las coordenadas:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="carga_imagen_2">
          <property name="text">
           <string>Conseguir Coordenadas</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>110</y>
       <width>371</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Configuraci칩n de la imagen</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Resumen</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckableComboBox</class>
   <extends>QComboBox</extends>
   <header>CheckableComboBox</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="color_picker.qrc"/>
  <include location="borde.qrc"/>
 </resources>
 <connections/>
</ui>
